src/SummaryRow.tsx

```diff
@@ -2,30 +2,29 @@ import { memo } from 'react';
 import clsx from 'clsx';
 import { css } from '@linaria/core';
 
-import { cell, row, rowClassname } from './style';
-import { getColSpan } from './utils';
+import { cell, row, rowClassname, rowSelectedClassname } from './style';
+import { getColSpan, getRowStyle } from './utils';
 import SummaryCell from './SummaryCell';
 import type { CalculatedColumn, RowRendererProps } from './types';
-import { useRovingRowRef } from './hooks';
 
 type SharedRowRendererProps<R, SR> = Pick<RowRendererProps<R, SR>, 'viewportColumns' | 'rowIdx'>;
 
 interface SummaryRowProps<R, SR> extends SharedRowRendererProps<R, SR> {
   'aria-rowindex': number;
   row: SR;
-  bottom: number;
+  top: number | undefined;
+  bottom: number | undefined;
   lastFrozenColumnIndex: number;
   selectedCellIdx: number | undefined;
   selectCell: (row: SR, column: CalculatedColumn<R, SR>) => void;
 }
 
 const summaryRow = css`
-  &.${row} {
+  z-index: 3;
```

Comment:
Does `z-index` actually work on `display: contents` elements? 🤔

Reply:
It turned out zIndex is not even needed as summary row is the last row